### PR TITLE
Roslyn Dev16.1 Preview 3 Snap

### PR DESCRIPTION
**Pull request by dpoeschl**

### CODE REVIEW

File: src/RoslynInsertionTool/scripts/VS-Insertion.ps1 [hunk 1]
```diff
@@ -59,10 +59,12 @@ Do-Insertion -component "Roslyn"            -queueName "Roslyn-Signed"         -
...
 #Do-Insertion -component "Live Unit Testing" -queueName "TestImpact-Signed"     -fromBranch "master"                   -toBranch "lab/d16.1stg" -insertCore "false"    -insertDevdiv "false"   -queueValidation "true"
 #Do-Insertion -component "VS Unit Testing"   -queueName "VSUnitTesting-Signed"  -fromBranch "dev16.1.x"                -toBranch "lab/d16.1stg" -insertCore "true"     -queueValidation "true" -dropPath "server"
 
 # Dev16.1 Preview 3
-Do-Insertion -component "Roslyn"            -queueName "Roslyn-Signed"         -fromBranch "master-vs-deps"           -toBranch "lab/d16.1stg" -insertToolset "true"  -insertDevdiv "false"   -updatecorextlibraries "true" -queueValidation "true"
 Do-Insertion -component "VS Unit Testing"   -queueName "VSUnitTesting-Signed"  -fromBranch "master"                   -toBranch "lab/d16.1stg" -insertCore "true"     -queueValidation "true" -dropPath "server"
+Do-Insertion -component "Roslyn"            -queueName "Roslyn-Signed"         -fromBranch "dev16.1-preview3-vs-deps" -toBranch "lab/d16.1stg" -insertToolset "true"  -insertDevdiv "false"   -updatecorextlibraries "true" -queueValidation "true"
```
Comment (dpoeschl) [inline src/RoslynInsertionTool/scripts/VS-Insertion.ps1]: Into `lab/d16.1stg` just until the divisional snap, when it'll change to `rel/d16.1`.

File: src/RoslynInsertionTool/scripts/VS-Insertion.ps1 [hunk 2]
```diff
@@ -59,10 +59,12 @@ Do-Insertion -component "Roslyn"            -queueName "Roslyn-Signed"         -
...
+
+# Dev 16.2 Preview 1
```
Comment (dpoeschl) [inline src/RoslynInsertionTool/scripts/VS-Insertion.ps1]: Not sure what's happening here.

**Comment by dpoeschl** [comment]
@RikkiGibson @jasonmalinowski Can you please take a look?

File: src/RoslynInsertionTool/scripts/VS-Insertion.ps1 (hunk 2 above)
Comment (RikkiGibson) [reply src/RoslynInsertionTool/scripts/VS-Insertion.ps1]: There's no code under there? I'm not sure either.

File: src/AzureFunctionPackage/Functions/Common/config.xml [hunk 3]
```diff
@@ -3,20 +3,20 @@
     <!-- Roslyn branches (non-vs-deps) (flowing between releases) -->
     <merge from="dev15.0.x" to="dev15.9.x" />
     <merge from="dev15.9.x" to="dev16.0" />
-    <merge from="dev16.0" to="dev16.1-preview2" />
-    <merge from="dev16.1-preview2" to="master" />
```
Comment (jasonmalinowski) [inline src/AzureFunctionPackage/Functions/Common/config.xml]: This should ideally be going dev16.0 into dev16.1-preview2 into dev16.1-preview3: the idea is that if we did a servicing fix into Preview 2, we don't want to ship Preview 3 without it.

File: src/RoslynInsertionTool/scripts/VS-Insertion.ps1 [hunk 4]
```diff
@@ -59,10 +59,12 @@ Do-Insertion -component "Roslyn"            -queueName "Roslyn-Signed"         -
 Do-Insertion -component "VS Unit Testing"   -queueName "VSUnitTesting-Signed"  -fromBranch "dev16.0"                  -toBranch "rel/d16.0"    -insertCore "true"     -queueValidation "true" -dropPath "server"
 
 # Dev16.1 Preview 2
-Do-Insertion -component "Roslyn"            -queueName "Roslyn-Signed"         -fromBranch "dev16.1-preview2-vs-deps" -toBranch "rel/d16.1"    -insertToolset "true"  -insertDevdiv "false"   -updatecorextlibraries "true" -queueValidation "true"
+#Do-Insertion -component "Roslyn"            -queueName "Roslyn-Signed"         -fromBranch "dev16.1-preview2-vs-deps" -toBranch "rel/d16.1"    -insertToolset "true"  -insertDevdiv "false"   -updatecorextlibraries "true" -queueValidation "true"
```
Comment (jasonmalinowski) [inline src/RoslynInsertionTool/scripts/VS-Insertion.ps1]: Should we just delete this section, given nothing in it now applies?

File: src/RoslynInsertionTool/scripts/VS-Insertion.ps1 (hunk 2 above)
Comment (jasonmalinowski) [reply src/RoslynInsertionTool/scripts/VS-Insertion.ps1]: Right now VS master is 16.2; it'll be snapping into d16.2stg in a few days.

File: src/AzureFunctionPackage/Functions/Common/config.xml (hunk 3 above)
Comment (dpoeschl) [reply src/AzureFunctionPackage/Functions/Common/config.xml]: I think I've fixed this. 16.0 -> 16.1P2 -> 16.1P3 -> 16.2

**Pull request merged**